src/achievements.hpp

```diff
@@ -107,4 +107,4 @@ class achievements
 	void process_achievements_file(const config& cfg, const std::string& content_source);
 
 	std::vector<achievement_group> achievement_list_;
-};
```

Comment:
this seems very unrelated to the PR

Reply:
Guessing it got picked up from the fix-whitespace script.